package.json

```diff
@@ -1814,7 +1814,7 @@
         "glob": "^7.1.2",
         "hash.js": "^1.1.7",
         "iconv-lite": "^0.4.21",
-        "inversify": "^4.11.1",
+        "inversify": "^5.0.1",
```

Comment:
No reason not to approve this. But I looked it up out of interested. Looks like this version is also basically two years old.  Looks like there was an update to the inversify js base this summer that has not made it to the npm package yet.

Reply:
From the commits doesn't look like much really changed. So probably not missing on much there.

Reply:
Yeah, I'm mainly interested in this change:
https://github.com/inversify/InversifyJS/blob/master/CHANGELOG.md#changed

> Change GUID to incremented counter for better performance #882